app/build.gradle

```diff
@@ -96,13 +96,16 @@ dependencies {
     implementation deps.third_party.commons_io
     implementation deps.third_party.commons_codec
     implementation deps.third_party.fastscroll
+    implementation deps.third_party.guava
```

Comment:
I agree that this may not be worth it, it's just what the Autofill sample code recommended. I did some research and would now want to switch to one of [Mozilla's Android Components](https://github.com/mozilla-mobile/android-components/blob/master/components/lib/publicsuffixlist/README.md). What do you think?

Reply:
Looks like it does all we need it to, and does that in a slim package. Please give it a try.

Reply:
```
> Task :app:minifyReleaseWithR8
R8: Missing class: java.lang.ClassValue
```

R8's already having a fit working through Guava's size. Would certainly love not needing it.

Reply:
I have completed the switch over to Mozilla's Public Suffix List and removed the Guava dependency.

I chose to reference the relevant Maven artifact with a `+` version. Although this should usually be avoided, it seemed reasonable to me for a library that is basically set in stone apart from a large list that is updated biweekly. If you don't like to have an unspecified version, feel free to fix one though, I did not give it much thought from the maintainer's perspective.